src/main/java/com/linecorp/armeria/common/thrift/text/StructContext.java

```diff
@@ -48,11 +55,14 @@
     // field.
     private final Map<String, TField> fieldNameMap;
 
+    private final Map<String, Class<? extends TEnum>> enumMap;
+
     /**
      * Build the name -> TField map
      */
     StructContext(JsonNode json) {
         super(json);
+        enumMap = new HashMap<>();
```

Comment:
I don't think there's a good constant estimate of how many fields of enum type there may be in a struct.